remote-wallet/src/remote_wallet.rs

```diff
@@ -250,65 +254,234 @@ pub struct RemoteWalletInfo {
     pub error: Option<RemoteWalletError>,
 }
 
-impl RemoteWalletInfo {
-    pub fn parse_path(path: String) -> Result<(Self, DerivationPath), RemoteWalletError> {
-        let wallet_path = Url::parse(&path).map_err(|e| {
-            Into::<RemoteWalletError>::into(DerivationPathError::InvalidDerivationPath(format!(
-                "parse error: {:?}",
-                e
-            )))
-        })?;
-
-        if wallet_path.host_str().is_none() {
-            return Err(DerivationPathError::InvalidDerivationPath(
-                "missing remote wallet type".to_string(),
-            )
-            .into());
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum Vendor {
```

Comment:
Why Vendor instead of Manufacturer here?

Reply:
Shorter and it's the "v" in USB's "VID".  Now that it's all in place, I can `sed` it to "Manufacturer" though

Reply:
Yeah, looks like it's typically "vendor id", but "manufacturer string". 🤷‍♀️ 
I actually don't feel strongly about this; mostly curious about your choice.

Reply:
I started with Vendor then went to plumb it in and thought, "damn, maybe no one will notice" :sweat_smile: